src/Evolution/Systems/NewtonianEuler/Characteristics.cpp

```diff
@@ -211,28 +211,26 @@ Matrix left_eigenvectors<1>(const tnsr::I<double, 1>& velocity,
          "Expected unit normal, but got normal with magnitude "
              << get(magnitude(unit_normal)));
 
-  // Temporary with a useful combination, as per Kulikovskii Ch3
-  const double theta = get(dot_product(velocity, velocity)) -
-                       get(specific_enthalpy) +
-                       get(sound_speed_squared) / get(kappa_over_density);
-
+  const double u2 = get(dot_product(velocity, velocity));
```

Comment:
`u` is one of the velocity components, so I'd just call this `velocity_squared` everywhere.